tests/foreman/api/test_role.py

```diff
@@ -1634,19 +1634,13 @@ def test_positive_access_all_global_entities_by_org_admin(self):
             verify=False
         )
         with self.assertNotRaises(HTTPError):
-            entities.Architecture(sc).search()
-            entities.Audit(sc).search()
-            entities.Bookmark(sc).search()
-            entities.CommonParameter(sc).search()
-            entities.DockerComputeResource(sc).search()
-            entities.LibvirtComputeResource(sc).search()
-            entities.OVirtComputeResource(sc).search()
-            entities.VMWareComputeResource(sc).search()
-            entities.ConfigGroup(sc).search()
-            entities.DockerHubContainer(sc).search()
-            entities.DockerRegistryContainer(sc).search()
-            entities.Errata(sc).search()
-            entities.OperatingSystem(sc).search()
+            for entity in [
+                entities.Architecture, entities.Audit, entities.Bookmark, entities.CommonParameter,
+                entities.DockerComputeResource, entities.LibvirtComputeResource,
```

Comment:
I agree with @pgagne here - it can be safely removed. Maybe DockerComputeResource does not return error code right now, but it might in the future. And when it does, we won't care. This is not something we want to be notified about, spend time understanding, debugging and finally changing test code. Remove it now and never look back at it.

Reply:
True